source/documentation/requirements.html.md

```diff
@@ -80,42 +83,106 @@ the CloudSIG using rdopkg:
           cloud7-openstack-ocata-release: python-eventlet-0.18.4-2.el7
           cloud7-openstack-ocata-testing: python-eventlet-0.18.4-2.el7
           cloud7-openstack-pike-testing: python-eventlet-0.20.1-2.el7
+          cloud7-openstack-pike-release: python-eventlet-0.20.1-2.el7
+          cloud7-openstack-queens-release: python-eventlet-0.20.1-2.el7
+          cloud7-openstack-queens-testing: python-eventlet-0.20.1-2.el7
+          cloud7-openstack-rocky-testing: python-eventlet-0.20.1-2.el7
+
         master-distgit: https://github.com/rdo-common/python-eventlet.git
         review-origin: null
         review-patches: null
         tags:
           dependency: null
-        maintainers: 
+        maintainers:
         - apevec@redhat.com
         - hguemar@fedoraproject.org
 
-    If the package is found, it's already in RDO repositories and no more actions are needed to add
-    it to the repos.
-    
+    Note that the version of the package included in repositories is given by the
+    CBS tags applied to each package (shown under buildsys-tags section for each package).
+    Tags have a format cloud7-openstack-&lt;release>-&lt;phase> where:
+
+    - release: is a the OpenStack release name, as pike, queens or rocky.
+    - phase:
+      - `candidate` phase is assigned to packages to be rebuilt in
+      CBS but not pushed to any RDO repository.
+      - `el7-build` (only available for Rocky and newer releases)
+      is assigned to packages that only required to build other
+      packages but not to deploy or run any OpenStack service.
+      - `testing` phase means that the package is used in deployments using RDO Trunk
+      repo and published in a testing repo, but not official CloudSIG repository.
+      - `release` phase means that is published in the official CloudSIG repository.
+      This phase is only available after a RDO version has been officially released
+      not for the one currently under development.
+
+    For example, the package included in cloud7-openstack-queens-release will published
+    in the [CloudSIG repo for queens](http://mirror.centos.org/centos-7/7/cloud/x86_64/openstack-queens/).
+
+    Note that, for the release currently under development (rocky right now), only the
+    testing phase will be available. The package included in cloud7-openstack-rocky-testing
+    will be the one used to deploy from RDO Trunk Master repositories and it will be
+    automatically pushed to cloud7-openstack-rocky-release at RDO Rocky is officially
+    released and published.
+
+    If the package is found for the required CBS tag, it's already in RDO repositories
+    and no more actions are needed to add it to the repos.
+
```

Comment:
We may want to add a simple flow to explain the flow: 

-candidate -> -testing -> -release
-candidate -> -el7-build

Reply:
It's kind of explained in the workflow https://github.com/amoralej/website/blob/d1bde77d996e0aa390db357e5cd95b234a1f0088/source/images/new-dependencies.png

Reply:
I've added the flow in second patch.